analysis_options.yaml

```diff
@@ -37,6 +38,7 @@ linter:
     - implementation_imports
     - invariant_booleans
     - iterable_contains_unrelated_type
+    - join_return_with_assignment
```

Comment:
this style just looks weird to me but :man_shrugging: if it cuts down on code review churn I guess ill take it